pipeline/src/transformers/eventDocument.ts

```diff
@@ -182,6 +184,9 @@ export const transformEventDocument = (
         type: "EventLocation",
         isOnline: !!isOnline,
       }),
+      isAvailableOnline: JSON.stringify({
```

Comment:
isAvailableOnline filter is 
```
isAvailableOnline: true || false
```
AggregatableValue is 
```
"isAvailableOnline": """{"isAvailableOnline":true}"""
```
This diverges from the common pattern of 
`"""{"type":"EventInterpretation","id":"Wn3STCoAACgAIedR","label":"Hearing loop"}"""` type/id/label
but is consistent with the structure of the `isAvailableOnline` property in the display object